tv_shows_app/src/styles/theme/components/card.ts

```diff
@@ -0,0 +1,76 @@
+import { createMultiStyleConfigHelpers } from '@chakra-ui/styled-system';
+import { headers } from 'next/headers';
+
+// This function creates a set of function that helps us create multipart component styles.
+const helpers = createMultiStyleConfigHelpers([
+  'card',
+  'header',
+  'body',
+  'container',
+]);
+
+const Card = helpers.defineMultiStyleConfig({
+  baseStyle: {
+    body: {
+      borderRadiusBotton: '8px',
+      overflow: 'hidden',
+    },
+    container: {
+      boxShadow: 'customShadow',
+      borderRadius: '8px',
+      overflow: 'hidden',
+    },
+  },
+  sizes: {
+    sm: {
+      card: {
+        width: '342px',
+        height: '480px',
+      },
+      container: {
+        margin: '32px',
```

Comment:
margine nikad necemo staviti u temu. Margine oznacavaju pozicioniranje elementa, to zelim raditi inline